src/common/interfaces.ts

```diff
@@ -1,4 +1,39 @@
+import { NewRasterLayer, UpdateRasterLayer } from '@map-colonies/mc-model-types';
+import { IJobResponse } from '@map-colonies/mc-priority-queue';
+
 export interface IConfig {
   get: <T>(setting: string) => T;
   has: (setting: string) => boolean;
 }
+
+export interface IQueueConfig {
+  jobManagerBaseUrl: string;
+  heartbeat: IHeartbeatConfig;
+  dequeueIntervalMs: number;
+  jobTypes: string[];
+  initTaskType: string;
+}
+
+export interface IHeartbeatConfig {
+  baseUrl: string;
+  intervalMs: number;
+}
+
+export interface LogContext {
+  fileName: string;
+  class?: string;
+  function?: string;
+}
+
+/* eslint-disable @typescript-eslint/naming-convention */
+export interface JobTypeMap {
```

Comment:
Is this interface necessary? not in use atm.
would like if you can explain it to me if it is.

Reply:
not in use, will be deleted

Reply:
forgot to remove.

Reply:
done